content/en/database_monitoring/setup_mysql/aurora.md

```diff
@@ -52,6 +52,7 @@ Configure the following in the [DB Parameter Group][3] and then **restart the se
 | --- | --- | --- |
 | `performance_schema` | `ON` | Required. Enables the [Performance Schema][1]. |
 | <code style="word-break:break-all;">performance_schema_consumer_events_statements_current</code> | `ON` | Required. Enables monitoring of currently running queries. |
+| <code style="word-break:break-all;">performance-schema-consumer-events-waits-current</code> | `ON` | Required. Enables the collection of wait events. |
```

Comment:
should these be underscores instead of dashes?

Reply:
Actually, nvm I see other tables have some with dashes.

Reply:
Good question, they should be dashes. I thought they should be underscores as well, but confirmed it should be dashes.